concent_api/verifier/tests/test_integration_verification.py

```diff
@@ -0,0 +1,123 @@
+import mock
+
+from django.test    import override_settings
+
+from golem_messages import message
+
+from conductor.models import BlenderSubtaskDefinition
+from core.tests.utils import ConcentIntegrationTestCase
+from utils.testing_helpers  import generate_ecc_key_pair
+from ..constants import VerificationResult
+from ..tasks        import blender_verification_order
+
+
+def mock_store_file_from_response_in_chunks_raise_exception(_response, _file_path):
+    raise Exception
+
+
+def mock_unpack_archive_raise_exception(_file_path):
+    raise OSError
+
+
+(CONCENT_PRIVATE_KEY, CONCENT_PUBLIC_KEY)                         = generate_ecc_key_pair()
+
+
+@override_settings(
+    CONCENT_PRIVATE_KEY         = CONCENT_PRIVATE_KEY,
+    CONCENT_PUBLIC_KEY          = CONCENT_PUBLIC_KEY,
```

Comment:
There shouldn't be so much space between parameter and value

Reply:
done :+1: